src/components/tiles/AddressTile2.tsx

```diff
@@ -40,6 +43,7 @@ interface OwnProps {
 }
 interface StateProps {
   account: EdgeAccount
+  currencyWallets: { [walletId: string]: EdgeCurrencyWallet }
```

Comment:
We already have the account, so just use `props.account.currencyWallets` instead of `props.currencyWallets`, and delete this.